core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -60,21 +60,27 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
               <span ng-if="getAverageRating()">
                 <[getAverageRating() | number:1]>
               </span>
-              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A">
+              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A" class="grayout">
```

Comment:
Classes added by us should be prefixed with .oppia to distinguish them from classes introduced by third-party libraries. See the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).

In addition, try to scope the CSS selector to this directive only, e.g. `.oppia-activity-summary-tile .oppia-inactive-summary-tile-attribute`. Note that I've picked a class name that explains what the thing is, rather than the particular styling applied to it -- you can choose a different class name, but keep the principle in mind.